modules/andes-core/broker/src/main/java/org/wso2/andes/store/cassandra/CQLBasedAndesContextStoreImpl.java

```diff
@@ -532,4 +532,41 @@ private ResultSet execute(Statement statement, String task) throws AndesExceptio
             throw new AndesException("Error occurred while " + task, e);
         }
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void updateExclusiveConsumerForQueue(String queueName, boolean isExclusiveConsumer) throws AndesException {
+
+        //Retrieving queue information
+        Statement statement1 = QueryBuilder.select().
+                column(QUEUE_EXCHANGE_NODE_DETAIL).
+                from(config.getKeyspace(), QUEUE_EXCHANGE_NODE_TABLE).
+                where(eq(QUEUE_NAME, queueName)).
+                setConsistencyLevel(config.getReadConsistencyLevel());
+
+        ResultSet resultSet = execute(statement1, "retrieve data of the queue :"+queueName);
+
+        String queueData = resultSet.toString();
+        String[] detailsOfQueue = queueData.split("=");
+
+        //Change the last value of the array with new exclusive consumer value
+        detailsOfQueue[detailsOfQueue.length-1] = String.valueOf(isExclusiveConsumer);
+
+        StringBuilder builder = new StringBuilder();
+        for(String s : detailsOfQueue) {
+            builder.append(s);
+        }
+        queueData = builder.toString();
+
+        Statement statement2 = QueryBuilder.update(config.getKeyspace(), QUEUE_EXCHANGE_NODE_TABLE).
```

Comment:
isn't it better to add meaningful varaible name rather than statement2 in here too?